### PR TITLE
all the functions has successfully tested

### PR DESCRIPTION
**Pull request by faridguluzade**

### CODE REVIEW

**Comment by faridguluzade** [comment]
I have done!

File: elnur.test.js
Comment (elnurastanov) [inline elnur.test.js:1]: Undo this file

**Review by elnurastanov — CHANGES_REQUESTED** [review]

File: farid.test.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  isEven,
+  isPrime,
+  reverseArray,
+  merge,
+  removeLastChar,
+  testStatement,
+  getLocalCurrency,
+} = require("./index");
+
+// Test isEven function
```
Comment (elnurastanov) [inline farid.test.js:11]: Remove description comments

**Review by elnurastanov — CHANGES_REQUESTED** [review]

File: farid.test.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    );
+  });
+
+  it("should throw error in different argument type", () => {
+    expect(() => merge({ firstName: "wolter" }, "white")).toThrow(
+      "Arguments must be object or array!"
+    );
+  });
+
+  it("should throw error in different argument type", () => {
```
Comment (elnurastanov) [inline farid.test.js:76]: Duplicated test case

File: farid.test.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  });
+
+  it("should return falsyValue argument the case of the statement is false", () => {
+    expect(testStatement(false, "Wolter", "Gustavo"));
+  });
+});
+
+// Test getLocalCurrency function
+describe.only("getLocalCurrency function", () => {
+  it("should return contry currency the case of the argument ", () => {
```
Comment (elnurastanov) [inline farid.test.js:129]: Where is naot found case?

**Review by elnurastanov — APPROVED** [review]

**Pull request merged**